src/main/java/app/attestation/server/AttestationServer.java

```diff
@@ -237,6 +238,7 @@ public static void main(final String[] args) throws Exception {
         server.createContext("/api/configuration", new ConfigurationHandler());
         server.createContext("/api/delete_device", new DeleteDeviceHandler());
         server.createContext("/api/devices.json", new DevicesHandler());
+        server.createContext("/api/attestation_history", new AttestationHistoryHandler());
```

Comment:
Since this is JSON, it makes sense to put `.json` at the end of the method name.